types/hooks/screens/useSignedChannelListScreenHook.types.ts

```diff
@@ -1,11 +1,12 @@
 import {ChannelList} from '../../database/schema/ChannelList.types';
+import {ContactScreenPayload} from './useChatUtilsHook.types';
 
 interface UseSignedChannelListScreenHook {
   channels: ChannelList[];
   goToChatScreen: (channel: ChannelList) => void;
   goToPostDetailScreen: (channel: ChannelList) => void;
   goToCommunityScreen: (channel: ChannelList) => void;
-  goToContactScreen: () => void;
+  goToContactScreen: ({from}: ContactScreenPayload) => void;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the signature of `goToContactScreen` function is a good improvement. It now expects an object with a `from` property as an argument, which makes the function more flexible and adaptable to different use cases.
```diff
-  goToContactScreen: () => void;
+  goToContactScreen: ({from}: ContactScreenPayload) => void;
```
This change should be reflected wherever this hook and specifically the `goToContactScreen` function is used.


<!-- This is an auto-generated comment by OSS CodeRabbit -->